README-SL.md

```diff
@@ -159,7 +157,6 @@ Povezave, ki neposredno usmerjajo na primerne `issues`, s katerim lahko prispeva
 ## Pobude za odprtokodne prispevke
 
 - [Up For Grabs](https://up-for-grabs.net/) - Vsebuje projekte z začetnikom prijaznimi težavami
-- [First Timers Only](https://www.firsttimersonly.com/) - Seznam hroščev, ki so označeni kot "samo za začetnike".
```

Comment:
This can be kept and added back in.

Reply:
why are we using this link for two time

Reply:
add66b602f87077fb9a28bfc9c24e09203be26b5 done